### PR TITLE
[RDAPI-25554] Adding customized properties for Amplify Connected Menu

### PR DESCRIPTION
**Pull request by dsantosaxway**
Customize Amplify Connected Menu for API Manager and API Gateway Manager

## Describe the changes

Adding customized properties details for Amplify Connected Menu.

## Deploy preview link

https://deploy-preview-2157--axway-open-docs.netlify.app/docs/apim_administration/apimgr_admin/api_mgmt_custom/#customize-amplify-menu-header

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [x] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

File: content/en/docs/apim_administration/apimgr_admin/api_mgmt_custom.md [hunk 1]
```diff
@@ -617,6 +617,53 @@ To specify the location of the API Gateway Manager console, perform the followin
 3. After updating the file, log into API Manager.
 4. Press **Ctrl-F5** to refresh.
 
+## Customize API Manager and API Gateway Manager Amplify Connected Menu
+
+1. Edit the following file:
+
+    ```
+    INSTALL_DIR/apigateway/webapps/emc/vordel/manager/app/app.config
```
Comment (dshielaxway) [inline content/en/docs/apim_administration/apimgr_admin/api_mgmt_custom.md:625]: This config file is specific to API Gateway Manager.
The API Manager equivalent is INSTALL_DIR/apigateway/webapps/apiportal/vordel/apiportal/app/app.config.
We should probably mention both.

**Review by kmercierwhite — APPROVED** [review]

File: content/en/docs/apim_administration/apimgr_admin/api_mgmt_custom.md (hunk 1 above)
Comment (dsantosaxway) [reply content/en/docs/apim_administration/apimgr_admin/api_mgmt_custom.md]: Added the path for both as suggested. Thanks!

**Review by dshielaxway — APPROVED** [review]

File: content/en/docs/apim_administration/apimgr_admin/api_mgmt_custom.md [hunk 2]
```diff
@@ -617,6 +617,53 @@ To specify the location of the API Gateway Manager console, perform the followin
...
+
+    API Manager: `INSTALL_DIR/apigateway/webapps/apiportal/vordel/apiportal/app/app.config`
+
+    API Gateway Manager: `INSTALL_DIR/apigateway/webapps/emc/vordel/manager/app/app.config`
+
+2. The menu is switched on by default, for example:
+
+    ```
+    connectedServicesMenu: {
+        headerOffline: false,
```
Comment (dshielaxway) [inline content/en/docs/apim_administration/apimgr_admin/api_mgmt_custom.md:632]: need to add in the 'host' field

host: "https://axway-portals-common-header.s3.eu-west-3.amazonaws.com",

The configuration for the old menu allowed customers to modify the host from which the menu was retrieved.

Comment (dsantosaxway) [reply content/en/docs/apim_administration/apimgr_admin/api_mgmt_custom.md:633]: Added the host as suggested. Thanks!

**Pull request merged**